locksmith/migrations/20230425123140-key-expiration-reminder.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('KeyExpirationReminders', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      lockAddress: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      network: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      tokenId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      type: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      expiration: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    })
+    await queryInterface.addIndex('KeyExpirationReminders', {
+      fields: ['id', 'lockAddress', 'network', 'tokenId', 'type', 'expiration'],
```

Comment:
I don't think it makes sense to add the `id` in there. 
When you create an index is it because you will use the indexed fields to make a query.
Here you will likely retrieve by network, lockAddress, tokenId, type and expiration, but NOT by `id`.